src/core/Table/Table.tsx

```diff
@@ -648,30 +656,59 @@ export const Table = <
             }
           }}
         >
-          {data.length !== 0 &&
-            page.map((row: Row<T>) => {
-              prepareRow(row);
-              return (
-                <TableRowMemoized
-                  row={row}
-                  rowProps={rowProps}
-                  isLast={row.index === data.length - 1}
-                  onRowInViewport={onRowInViewportRef}
-                  onBottomReached={onBottomReachedRef}
-                  intersectionMargin={intersectionMargin}
-                  state={state}
-                  key={row.getRowProps().key}
-                  onClick={onRowClickHandler}
-                  subComponent={subComponent}
-                  isDisabled={!!isRowDisabled?.(row.original)}
-                  tableHasSubRows={hasAnySubRows}
-                  tableInstance={instance}
-                  expanderCell={expanderCell}
-                />
-              );
-            })}
+          {data.length !== 0 && (
```

Comment:
Try scrolling with keyboard (arrows, page up/down), it stop scrolling after few items. As I understand the problem is that focused row disappears and keyboard actions stop working. Workaround could be adding `tabIndex` to the table body.

Reply:
ok.. So added your suggestion.
Found only one issue: when you have table with selectable rows, select one row and press arrow up/down, seems like a swap in row or something like that :D But I think its minor, compared that previously it was not working at all...